Zend/zend_attributes.c

```diff
@@ -55,6 +56,13 @@ void validate_attribute(zend_attribute *attr, uint32_t target, zend_class_entry
 	}
 }
 
+void validate_return_type_will_change_attribute(zend_attribute *attr, uint32_t target, zend_class_entry *scope)
+{
+	if (target != ZEND_ATTRIBUTE_TARGET_METHOD) {
+		zend_error(E_COMPILE_ERROR, "Only methods can be marked with #[ReturnTypeWillChange]");
```

Comment:
Sounded like a good idea, however this code doesn't do anything because:
- for userland code, `zend_compile_attributes()` validates the target before `validate_return_type_will_change_attribute ()` is reaxhed
- that's why I added the attribute to a new internal class in zend_test, but validation doesn't happen anywhere (or am I expected to invoke the validator manually?)

So I should remove this piece of code, shouldn't I?